completions/bash-nvme-completion.sh

```diff
@@ -884,6 +884,9 @@ plugin_micron_opts () {
 		"vs-smbus-option")
 		opts+=" --option= -o --value= -v --save= -s"
 			;;
+		"ocp-telemetry-log-parse")
```

Comment:
The zsh completions _nvme file is not added the command.

Reply:
Still working on this item.

Reply:
Added zsh completions _nvme file.